### PR TITLE
fix: some typos in comments

### PR DESCRIPTION
**Pull request by flanagansteve**
- typo on overstimate in _getSolvencyBalances - Found this comment while tracing check solvency logic
- miscapitalisation of For in subRect - should be For not FOr
- (from Henry) remove claim in PanopticFactory about the SFPM that is no longer true

### CODE REVIEW

File: contracts/PanopticPool.sol [hunk 1]
```diff
@@ -168,7 +168,7 @@ contract PanopticPool is ERC1155Holder, Multicall {
     SemiFungiblePositionManager internal immutable SFPM;
 
     /*//////////////////////////////////////////////////////////////
-                                STORAGE 
+                                STORAGE
```
Comment (flanagansteve) [inline contracts/PanopticPool.sol:171]: (this change seems fine too, i think an automatic linter in my pre-commit made it. happy to revert this change though the typo is what i care more about)

**Review by PeakyCryptos — APPROVED** [review]

**Comment by dyedm1** [comment]
Tests flaked - rerunning

**Comment by dyedm1** [comment]
See #89, rerunning tests again

**Review by dyedm1 — APPROVED** [review]
